### DISCUSSION TITLE
select * from jsonb_each_text(jsonb)

### DISCUSSION THREAD

**Discussion by boYu07** [go-jet/jet#250] [Q&A]
I couldn't find a way to assemble the title query string, any suggestion?

**Comment by go-jet** [comment]
Hi @boYu07 ,
Json functions are not supported currently. Can't think of workaround for this case unless [raw statement](https://github.com/go-jet/jet/wiki/Statements#raw-statements) is used.

**Reply by boYu07** [reply]
i tried raw statement

```golang
func QueryJsonbEachText(tab postgres.Table, column postgres.Column) postgres.Statement {
	quertString := `SELECT key, value FROM $table, jsonb_each_text($column)`
	s := postgres.RawStatement(quertString, postgres.RawArgs{"$table": tab.TableName(), "$column": column.Name()})
	return s
}

func main() {
	column := postgres.StringColumn("test_column") // this is the json column
	tab := postgres.NewTable("", "test_table", "test_table", column)
	s := QueryJsonbEachText(tab, column)
	fmt.Println(s.DebugSql())
}
```
i expect the query string like this:
`SELECT key, value FROM test_table, jsonb_each_text(test_column);`
but i got
`SELECT key, value FROM 'test_table', jsonb_each_text('test_column');`

**Comment by go-jet** [comment]
There is no need for `RawArgs` in your case. `RawArgs`  are placeholder parameters for the query, and can't be used for table source. 
```golang
stmt := postgres.RawStatement(`SELECT key, value FROM test_table, jsonb_each_text(test_column)`)
```

**Reply by boYu07** [reply]
I figure it out, thanks.

**Comment by go-jet** [comment]
With Release v.2.12.0 there is partial support for set returning functions - [faq](https://github.com/go-jet/jet/wiki/FAQ#set-returning-functions). The whole statement doesn't have to be raw statement.
